packages/api/test/user.spec.js

```diff
@@ -5,46 +5,18 @@ import { endpoint } from './scripts/constants.js'
 import { getTestJWT, getDBClient } from './scripts/helpers.js'
 import userUploads from './fixtures/pgrest/get-user-uploads.js'
 
-describe('GET /user/account', () => {
-  it('error if not authenticated with magic.link', async () => {
```

Comment:
is there a reason you're removing these tests?

Reply:
we're basically merging the `account` and `info` endpoints so these tests are now redundant with the `info` tests.  though i suppose i could copy the asserts over to prove that functionality still works.

Reply:
fixed: https://github.com/web3-storage/web3.storage/pull/1600/commits/c23632f3d8f84cdceaf8b19108f7cadb9bce56d4#diff-fd06e8f47f44010478243d794d7e859dc226dbda62aa8796511d5b86bc00d207R38